### DISCUSSION TITLE
Windows VM unusable on Apple M2 Host

### DISCUSSION THREAD

**Discussion by ibboaz** [utmapp/UTM#4706] [UTM Help (macOS)] [answered]
**Describe the issue**  
Windows host unusable - no network, no clipboard access, no shared drive access.  Installation proceeded without problem.  Spice guest tools installation proceeded without problem.  Multiple reboots to no effect.  Shared drive was populated but no Z: drive appeared on guest (or anything else that would be a 'shared' drive).

**Configuration**  
* UTM Version: 4.0.9
* macOS Version: 12.6.1
* Mac Chip (Intel, M1, ...): Apple M2
* Windows 7 - 64 bit guest
* Spice tools properly installed
* rebooted guest and host multiple times 

**Crash log**  
No crashes.

**Comment by osy** [comment, marked as answer]
For Windows 7, try changing your network adapter hardware to `rtl8139`. I don't think Windows 7 includes the e1000 driver by default.
